### PR TITLE
Phoenix/apirouter

### PR DESCRIPTION
**Pull request by phoeN24**
completed:
-api routers getMusic, getPodcasts

### CODE REVIEW

**Review by jchung07 — APPROVED** [review]
Looks good.

File: server/router/api.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//import controllers
+const userController = require('../controller/userController')
+const apiController = require('../controller/apiController')
+const express = require('express');
+const router = express.Router();
+
+
+//get user
+router.get('/',
```
Comment (youngmineeh) [inline server/router/api.js]: Does this need to be a post handler?

**Review by youngmineeh — APPROVED** [review]
Change looks good

**Comment by phoeN24** [comment]
do not merge, its linked to main not dev

**Pull request closed without merging**